rbac/policy.csv

```diff
@@ -18,3 +18,7 @@ p, admin, websocket, /ws/flow, allow
 p, admin, websocket, /ws/publisher, allow
 p, admin, websocket, /ws/replication, allow
 p, admin, websocket, /ws/subscriber, allow
+p, admin, topology/node, read, allow
```

Comment:
only the name of the resource is used, so it should be
```
p, admin, node, read, allow
p, admin, node, write, allow
p, admin, edge, read, allow
p, admin, edge, write, allow
```

Reply:
i am using the resource name as topology/node and topology/edge

Reply:
```
p, admin, noderule, read, allow
p, admin, edgerule, read, allow
```